.github/workflows/pr.yml

```diff
@@ -83,6 +83,11 @@ jobs:
           popd
           popd
 
+          #!/bin/bash
+
+          echo "Add serve target to nx next app"
+          ./add-serve-target.sh -w ../$NX_WORKSPACE -a $NX_APP
+
```

Comment:
Testing the plugin involves generating a default nx next app, and generating and running the e2e app against it. The generated next app no longer has a "serve" target which is expected by the e2e app, only a "serveStatic" target. 
This script adds this serve target, so that the e2e app can reference it and the test pass.